workers/data_refinery_workers/processors/agilent_twocolor.py

```diff
@@ -106,8 +106,15 @@ def _create_result_objects(job_context: Dict) -> Dict:
     result.time_start = job_context['time_start']
     result.time_end = job_context['time_end']
     result.pipeline = "Agilent SCAN TwoColor"  # TODO: should be removed
-    # result.processor = Processor.objects.get(name=utils.ProcessorEnum.AGILENT_TWOCOLOR.value,
-    #                                          version = __version__)
+    try:
+        result.processor = utils.find_processor("AGILENT_TWOCOLOR")
+    except Exception as e:
+        err_str = "Failed to set processor: %s" % e
+        logger.error(err_str)
```

Comment:
Could you add some additional context to this log message. Maybe like `job_id=job_context["job"].id, processor="AGILENT_TWOCOLOR"`?